package.json

```diff
@@ -85,7 +85,7 @@
       "prettier --write",
       "git add"
     ],
-    "*.json": [
+    "*.(json|md)": [
```

Comment:
pre commit hook did not check markdown files but github action does.
so CI failed if someone edit markdown file and did not run prettier.